common/OpTestIPMI.py

```diff
@@ -1075,10 +1076,7 @@ def ipmi_sol_activate(self):
     #
     def ipmi_get_console(self):
         self.ipmi_sol_deactivate()
-        # Waiting for a small time interval as latter versions of ipmi takes a bit of time to deactivate.
-        time.sleep(BMC_CONST.IPMI_SOL_DEACTIVATE_TIME)
         l_con = self.ipmi_sol_activate()
-        time.sleep(BMC_CONST.IPMI_SOL_ACTIVATE_TIME)
         count = 0
```

Comment:
Agreed, that likely makes it a lot more solid against BMCs that aren't always agreeable.
The commit I just pushed addresses this.